docs/docs/resources/pipeline.yaml

```diff
@@ -0,0 +1,334 @@
+        valueSchema: value-schema
+        partitions_count: 1
+        replication_factor: 1
+        configs: 
+          config1: config1
+    models: 
+      model: model
+  # Pipeline prefix that will prefix every component name. If you wish to not 
+  # have any prefix you can specify an empty string.
+  prefix: ${pipeline_name}-
+  # Helm repository configuration, the default value is given here
+  repoConfig:
+    repositoryName: bakdata-streams-bootstrap # required
+    url: https://bakdata.github.io/streams-bootstrap/ # required
+    repo_auth_flags:
+      username: user
+      password: pass
+      ca_file: /home/user/path/to/ca-file
+      insecure_skip_tls_verify: false
+  version: "2.7.0"
+
+# Kafka source connector
+- type: kafka-source-connector
+  name: kafka-source-connector # required
+  namespace: namespace # required
+  # `app` contains application-specific settings, hence it does not have a rigid
+  # structure. The fields below are just an example.
+  app: # required
+    key1:
+      key4: value3
+    key2: value1
+    key3: value2
+  # Topic(s) from which the component will read input
+  from:
+    topics: # required
+      ${pipeline_name}-input-topic:
+        type: input # required
+        role: topic-role
+  # Topic(s) into which the component will write output
+  to:
+    topics: # required
+      ${pipeline_name}-output-topic:
+        type: output # required
+        role: topic-role
+        keySchema: key-schema
+        valueSchema: value-schema
+        partitions_count: 1
+        replication_factor: 1
+        configs: 
+          config1: config1
+    models: 
+      model: model
+  # Pipeline prefix that will prefix every component name. If you wish to not 
+  # have any prefix you can specify an empty string.
+  prefix: ${pipeline_name}-
+  # Helm repository configuration
+  repoConfig:
+    repositoryName: my-repo # required
+    url: https://bakdata.github.io/kafka-connect-resetter/ # required
+    repoAuthFlags: 
+      username: user
+      password: pass
+      ca_file: /home/user/path/to/ca-file
+      insecure_skip_tls_verify: false
+  version: "1.0.4"
+  # Overriding Kafka Connect Resetter Helm values. E.g. to override the
+  # Image Tag etc.
+  resetterValues:
+    imageTag: 1.2.3
+  offsetTopic: offset_topic
+    
+- type: kafka-sink-connector
+  name: kafka-sink-connector # required
+  namespace: namespace # required
+  # `app` contains application-specific settings, hence it does not have a rigid
+  # structure. The fields below are just an example.
+  app: # required
+    key1:
+      key4: value3
+    key2: value1
+    key3: value2
+  # Topic(s) from which the component will read input
+  from:
+    topics: # required
+      ${pipeline_name}-input-topic:
+        type: input # required
+        role: topic-role
+  # Topic(s) into which the component will write output
+  to:
+    topics: # required
+      ${pipeline_name}-output-topic:
+        type: output # required
```

Comment:
Only error topics for sink connectors

Reply:
Hm, the schema says that the `to` section is the same as the one for `kubernetesApp`. Is the user required to specify the type at all?

Reply:
If the user specifies an output topic for sinks, it will just be ignored. So in order to not confuse anyone, we should only use error topics for sinks